src/Simplex/Chat/Messages.hs

```diff
@@ -790,6 +796,7 @@ ciRequiresAttention content = case msgDirection @d of
     CIRcvChatFeatureRejected _ -> True
     CIRcvGroupFeatureRejected _ -> True
     CIRcvModerated -> True
+    _ -> False
```

Comment:
I think it's better to keep this case complete to remember to make a decision for new chat item types, instead of guessing at unexpected effects later

```suggestion
    CIInvalidJSON _ -> False
```